plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.cpp

```diff
@@ -0,0 +1,28 @@
+/* -*- mode: c++ -*-
+ * Copyright (C) 2017, 2018  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+//
+//#include <Kaleidoscope-Devel-ArduinoTrace.h>
+//
+//namespace kaleidoscope {
+//namespace plugin {
+//
+//EventHandlerResult ArduinoTrace::onSetup() {
+//  return EventHandlerResult::OK;
+//}
+//}
+//}
+//
+//kaleidoscope::plugin::ArduinoTrace ArduinoTrace;
```

Comment:
If the plugin doesn't provide anything else other than pulling ArduinoTrace in, do we even need thtis file?